local_pathfinding/local_path.py

```diff
@@ -7,13 +7,33 @@
 
 
 class LocalPathState:
+    """Gathers and stores the state of the Sailbot.
```

Comment:
Instead of copying units and conventions, I would add a sentence something like this:

The attributes' units and conventions can be found in the msgs they are derived from in the custom_interfaces repository.